### PR TITLE
Add a volume in pods with nginx sidecar to share fs

### PR DESCRIPTION
**Pull request by drgarcia1986**
This change allows nginx to serves application static content

(depends on https://github.com/luizalabs/slugrunner/pull/6)

### CODE REVIEW

File: pkg/server/spec/deploy.go [hunk 1]
```diff
@@ -87,7 +87,8 @@ func NewDeploy(imgs *Images, description, slugURL string, rhl int, a *app.App, t
 		fs,
 	)
 	ps.Containers[0].Args = []string{"start", a.ProcessType}
-	ps.Containers[0].VolumeMounts = []*VolumeMounts{newSlugVolumeMount()}
+	ps.Containers[0].VolumeMounts = append(ps.Containers[0].VolumeMounts, newSlugVolumeMount())
```
Comment (yagonobre) [inline pkg/server/spec/deploy.go]: Why change this?

File: pkg/server/spec/pod.go [hunk 2]
```diff
@@ -55,6 +57,12 @@ func newPodContainers(name, nginxImage, appImage string, envVars map[string]stri
 		newAppContainer(name, appImage, envVars, appPort, secrets),
 	}
 	if nginxImage != "" {
+		c[0].VolumeMounts = []*VolumeMounts{
```
Comment (yagonobre) [inline pkg/server/spec/pod.go]: Mixed feelings about that, maybe this can be moved to `newAppContainer`.

File: pkg/server/spec/deploy.go (hunk 1 above)
Comment (drgarcia1986) [reply pkg/server/spec/deploy.go]: the old way just ignore any volume mounts created before

File: pkg/server/spec/pod.go (hunk 2 above)
Comment (drgarcia1986) [reply pkg/server/spec/pod.go]: `newAppContainer` func doesn't know about nginx right?

Comment (yagonobre) [reply pkg/server/spec/pod.go]: `newAppContainer` receive `secondaryPort` as `port` value, but I'm not sure if it's a good approach

Comment (drgarcia1986) [reply pkg/server/spec/pod.go]: hm, I'm lost with tons of doubts LOL, @aguerra what do you think about it?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - Return error with timeout message when there's no resources for build, release phase and command exec
 
+### Added
+- Share application container file system (directory `/app`) with nginx sidecar
+
```
[the remark this answers is not in the document]
Comment (aguerra) [reply CHANGELOG.md]: I guess  is shorter and to the point

File: pkg/server/spec/pod.go (hunk 2 above)
Comment (aguerra) [reply pkg/server/spec/pod.go]: :shipit:

**Pull request merged**